Makefile

```diff
@@ -16,6 +16,7 @@ PG_AUTH_MON?=v2.0
 PG_STAT_MONITOR?=2.0.3
 PG_LOGERRORS?=18d9795
 PGVECTOR?=v0.5.1
+PGVECTO_RS?=0.1.11
```

Comment:
Since 0.1.13 was just released, I'd suggest updating to that one and testing. Might as well start with the latest!

Reply:
Absolutely. This was released after I created the PR, but the PR is now updated to version `0.1.13`. Thanks for the hint to the latest release.